lokinet/contrib/lokinet-auth.ini

```diff
@@ -1,3 +1,4 @@
+#What is this ip?
```

Comment:
please make separate issues for each of these questions instead of in a git commit on a PR so they are tracked and we can close them as they get addressed.

Reply:
Sounds good! I figured out a number of things on my own but I'm not sure how to close the PR - Please do if you can.